esphome/components/esp32/gpio.cpp

```diff
@@ -54,13 +54,13 @@ struct ISRPinArg {
 
 ISRInternalGPIOPin ESP32InternalGPIOPin::to_isr() const {
   auto *arg = new ISRPinArg{};  // NOLINT(cppcoreguidelines-owning-memory)
-  arg->pin = this->pin_;
+  arg->pin = this->get_pin_num();
   arg->flags = gpio::FLAG_NONE;
-  arg->inverted = inverted_;
+  arg->inverted = pin_flags_.inverted;
```

Comment:
> All uses of class members and member functions should be prefixed with this-> to distinguish them from global functions/variables.

https://developers.esphome.io/contributing/code/#c-code-style
```suggestion
  arg->inverted = this->pin_flags_.inverted;
```